client/components/Overview/Overview.jsx

```diff
@@ -88,19 +90,23 @@ class Overview extends React.Component {
       var price;
       var salePrice;
       var styleItem = currentItem[this.state.index].name;
+      var thumbOrIndex;
+      var url;
+
       if(currentItem[this.state.index].sale_price !== null) {
```

Comment:
I think taking this condition logic out of the return block, instead of nesting it inside, actually makes it easier to read and follow. This is great! I'm curious what's your decision behind passing HTML elements as props to child components.